linear_algebra/src/power_iteration.py

```diff
@@ -0,0 +1,96 @@
+import numpy as np
+
+
+def power_iteration(
+    input_matrix: np.array, vector: np.array, error_tol=1e-12, max_iterations=100
+) -> [float, np.array]:
+
+    """
+    Power Iteration.
+    Find the largest eignevalue and corresponding eigenvector
+    of matrix input_matrix given a random vector in the same space.
+    Will work so long as vector has component of largest eigenvector.
+    input_matrix must be symmetric.
+
+    Input
+    input_matrix: input matrix whose largest eigenvalue we will find.
+    Numpy array. np.shape(input_matrix) == (N,N).
+    vector: random initial vector in same space as matrix.
+    Numpy array. np.shape(vector) == (N,) or (N,1)
+
+    Output
+    largest_eigenvalue: largest eigenvalue of the matrix input_matrix.
+    Float. Scalar.
+    largest_eigenvector: eigenvector corresponding to largest_eigenvalue.
+    Numpy array. np.shape(largest_eigenvector) == (N,) or (N,1).
+
+    >>> import numpy as np
+    >>> A = np.array([
```

Comment:
Please use the same variable names as the function parameters so the reader does not get lost.

Reply:
Thanks @cclauss this is a great point; I made the change :)